CHANGES.txt

```diff
@@ -112,6 +112,11 @@ Changes
 Fixes
 =====
 
+- Handle ``null`` values correctly in the cluster tab of the Admin UI.
```

Comment:
Please rephrase to use past tense. eg: Fixed handling of ``null`` values in the cluster tab...